DetectorDescription/DDCMS/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
 <use   name="FWCore/Utilities"/>
 <use   name="dd4hep"/>
 <use   name="rootmath"/>
+<use   name="rooteve"/>
```

Comment:
@smuzaffar - I'm not sure we want to depend on Eve. I can check it next year.

Reply:
I agree with Yana.

Reply:
I think we need a new root package which has as its top `libGeo`.

Reply:
PR https://github.com/cms-sw/cmssw/pull/25566 should resolve it.